boa/src/builtins/string/mod.rs

```diff
@@ -242,26 +244,30 @@ impl String {
     pub(crate) fn char_at(this: &Value, args: &[Value], context: &mut Context) -> Result<Value> {
         // First we get it the actual string a private field stored on the object only the context has access to.
         // Then we convert it into a Rust String by wrapping it in from_value
```

Comment:
We should remove them from all of the methods since it's from when we had the `from_value` function. The comment is deprecated, and does not have any purpose why it's there.

Reply:
Thanks for pointing it out. It reminds me to check the outdated comments, and now they are all removed.